<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PicView</class>
 <widget class="QDialog" name="PicView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>907</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图片浏览</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>631</width>
     <height>811</height>
    </rect>
   </property>
   <widget class="QLabel" name="labelPic">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonDownload">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>860</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>下载</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>860</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
